package/src/prmon.cpp

```diff
@@ -263,10 +278,11 @@ int main(int argc, char* argv[]) {
       {"netdev", required_argument, NULL, 'n'},
       {"help", no_argument, NULL, 'h'},
       {"level", required_argument, NULL, 'l'},
+      {"fast-memmon", no_argument, NULL, 'm'},
       {0, 0, 0, 0}};
 
   int c;
-  while ((c = getopt_long(argc, argv, "-p:f:j:i:d:sun:h:l:", long_options,
+  while ((c = getopt_long(argc, argv, "-p:f:j:i:d:sun:h:l:m:", long_options,
```

Comment:
Should this be `"-p:f:j:i:d:sun:h:l:m"`? The `-m` option doesn't take an argument, it's a toggle.

Reply:
Yep, done now 👍